src/PhpWord/ComplexType/RubyProperties.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/**
+ * This file is part of PHPWord - A pure PHP library for reading and writing
+ * word processing documents.
+ *
+ * PHPWord is free software distributed under the terms of the GNU Lesser
+ * General Public License version 3 as published by the Free Software Foundation.
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code. For the full list of
+ * contributors, visit https://github.com/PHPOffice/PHPWord/contributors.
+ *
+ * @see         https://github.com/PHPOffice/PHPWord
+ *
+ * @license     http://www.gnu.org/licenses/lgpl.txt LGPL version 3
+ */
+
+namespace PhpOffice\PhpWord\ComplexType;
+
+use InvalidArgumentException;
+
+/**
+ * Ruby properties.
+ *
+ * @see https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.wordprocessing.rubyproperties?view=openxml-3.0.1
+ */
+class RubyProperties
```

Comment:
@Deadpikle It misses unit tests for RubyProperties.

Reply:
Oops, sorry about that. I've added some unit tests for `RubyProperties`. 😄